docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -196,3 +200,8 @@ and a file system location of ``/var/my_tables/test_table``::
         format = 'PARQUET',
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
+
+Iceberg limitations
+-------------------
+
+Iceberg cannot be used to create views. Details `here <https://github.com/trinodb/trino/blob/e4cae791c0b56509af281ceb7b91bc7c62dce6e5/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java#L1035>`_.
```

Comment:
We generally don't link to the code in the documentation
It is sufficient to say that `Iceberg does not support creation of VIEWs`

Reply:
Agreed. Also dont use "here" for links. Along these lines we will work towards a doc PR to add a "SQL support" section in the connector docs that detail what actually is supported.

Reply:
Updated the text